### DISCUSSION TITLE
Use Git's own orange icon for Git for Windows

### DISCUSSION THREAD

**Discussion by ivadham** [git-for-windows/git#4477] [Ideas]
The utilization of a modified icon to indicate that this is a fork of Git containing Windows-specific patches may appear logical; nonetheless, it has a few drawbacks. Specifically, due to low contrast, notably in the yellow region, the icon becomes arduous to distinguish in the context menu. Additionally, given that this Git variant runs on Windows, it is conspicuous that it is Git for Windows.

Alternatively, if using the orange color isn't an option, Microsoft’s signature blue color for Windows can become a distinctive cue that this is Git for Windows.

Also, to further integrate seamlessly, modifying the verbiage can be considered:
`Git GUI Here` to `Open Git GUI here`. (Note that `here` is now in lowercase)
`Git Bash Here` to `Open Git Bash here`. (Note that `here` is now in lowercase)

![Git for Windows](https://github.com/git-for-windows/git/assets/11696957/77897b6e-b73f-4eec-afb5-076776319762)

**Comment by dscho** [comment]
I am afraid that brand recognition has been built for almost 16 years now, I would need a lot of data to support a switch to a different icon.

About the menu item labels: good idea! They are [defined here](https://github.com/git-for-windows/build-extra/blob/833475c533087245c5bdbde3400f1c549ebe7f27/installer/install.iss#L108-L109) and it should be a really easy pull request to get in.
